buster_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/codahale/buster"
+	"github.com/picmonkey/buster"
```

Comment:
Yeah, sorry, wrong repo.  I'm sure you can see the idea here though: rate is an int but is immediately converted to a float64.  Making it a float gives more precision and allows for trickle testing (rate < 1).  I'll send you a better PR for this unless you don't want it at all :grinning: